### PR TITLE
Added tags for clustering

### PR DESCRIPTION
**Pull request by aryanpola**
What does this implement/fix? Explain your changes.
I've added the necessary tags for some files under clustering.
#2035

### CODE REVIEW

**Comment by aryanpola** [comment]
@chrisholder I've run the pre-commit and showed no errors. Can you help me with the errors.

**Comment by TonyBagnall** [comment]
thanks for this, but it not what quite what I meant. If you look at classification,  we have a tag on the type of algorithm, e.g

"algorithm_type": "distance"

or

"algorithm_type": "deeplearning",

this is the only tag we want adding, not all the constructor arguments.

**Comment by aryanpola** [comment]
> thanks for this, but it not what quite what I meant. If you look at classification, we have a tag on the type of algorithm, e.g
> 
> "algorithm_type": "distance"
> 
> or
> 
> "algorithm_type": "deeplearning",
> 
> this is the only tag we want adding, not all the constructor arguments.

@TonyBagnall I've made changes, please let me know if there's more to do.

**Review by TonyBagnall — CHANGES_REQUESTED** [review]
thanks for this, minor things
1. could you change KShape to just "distance"
2. You do not need to repeat the tag for classes that inherit from KMedoids, you can just remove them for CLARANS and CLARA

**Comment by aryanpola** [comment]
> thanks for this, minor things
> 
> 1. could you change KShape to just "distance"
> 2. You do not need to repeat the tag for classes that inherit from KMedoids, you can just remove them for CLARANS and CLARA

I've changed them. Thanks for the help😊

File: aeon/clustering/base.py [hunk 1]
```diff
@@ -22,6 +22,10 @@ class BaseClusterer(BaseCollectionEstimator, ABC):
         Number of clusters for model.
     """
 
+    _tags = {
+        "algorithm_type": "distance",
+    }
+
```
Comment (TonyBagnall) [inline aeon/clustering/base.py:28]: dont want this

File: aeon/clustering/_kernel_k_means.py [hunk 2]
```diff
@@ -87,6 +87,7 @@ class TimeSeriesKernelKMeans(BaseClusterer):
     _tags = {
         "capability:multivariate": True,
         "python_dependencies": "tslearn",
+        "algorithm_type": "kernel-k-means",
```
Comment (TonyBagnall) [inline aeon/clustering/_kernel_k_means.py:90]: can you just remove this? I dont want to introduce a new type here

**Review by TonyBagnall — CHANGES_REQUESTED** [review]

**Review by TonyBagnall — APPROVED** [review]

**Comment by TonyBagnall** [comment]
thanks for this

**Pull request merged**